diff --git a/dist/components/SortableContext.d.ts b/dist/components/SortableContext.d.ts
index aa0521ee9c9aa5d2ca027375c6bd86ad8e504bef..eee54b9793cbf54a6c6197be21d31dc598ce2845 100644
--- a/dist/components/SortableContext.d.ts
+++ b/dist/components/SortableContext.d.ts
@@ -22,5 +22,5 @@ interface ContextDescriptor {
     strategy: SortingStrategy;
 }
 export declare const Context: React.Context<ContextDescriptor>;
-export declare function SortableContext({ children, id, items: userDefinedItems, strategy, disabled: disabledProp, }: Props): JSX.Element;
+export declare function SortableContext({ children, id, items: userDefinedItems, strategy, disabled: disabledProp, }: Props): React.JSX.Element;
 export {};
